src/ecdsa/keys.py

```diff
@@ -263,10 +263,20 @@ def precompute(self, lazy=False):
            use (when set to True)
         """
         if isinstance(self.curve.curve, CurveEdTw):
-            return
-        self.pubkey.point = ellipticcurve.PointJacobi.from_affine(
-            self.pubkey.point, True
-        )
+            pt = self.pubkey.point
+            self.pubkey.point = ellipticcurve.PointEdwards(
```

Comment:
Not a fan of precomputing through a setter, but seems like you've made your mind on it.

Reply:
yes, as it also allows changing the implementation